src/components/Reports/StaffExpenseReport/Tables/PrintTables.test.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import { ThemeProvider } from '@mui/material/styles';
+import { LocalizationProvider } from '@mui/x-date-pickers';
+import { AdapterLuxon } from '@mui/x-date-pickers/AdapterLuxon';
+import { render } from '@testing-library/react';
+import { GqlMockedProvider } from '__tests__/util/graphqlMocking';
+import theme from 'src/theme';
+import { ReportsStaffExpensesQuery } from '../GetStaffExpense.generated';
+import { TableType } from '../Helpers/StaffReportEnum';
+import { PrintTables, PrintTablesProps } from './PrintTables';
+
+const mutationSpy = jest.fn();
+
+interface TestComponentProps {
+  tableProps?: Partial<PrintTablesProps>;
+}
+
+const TestComponent: React.FC<TestComponentProps> = ({ tableProps }) => (
+  <LocalizationProvider dateAdapter={AdapterLuxon}>
+    <ThemeProvider theme={theme}>
+      <GqlMockedProvider<{
+        StaffExpenseReport: ReportsStaffExpensesQuery;
+      }>
+        mocks={{
+          reportsStaffExpenses: {
+            accountId: 'account-1',
+            name: 'Test Account',
+            status: 'active',
+            startBalance: 1000,
+            endBalance: 2000,
+            funds: [
+              {
+                fundType: 'Primary',
+                total: -500,
+                categories: [
+                  {
+                    category: 'Travel',
+                    total: -300,
+                    averagePerMonth: -100,
+                    subcategories: [
+                      {
+                        subCategory: 'Flights',
+                        total: -200,
+                        averagePerMonth: -50,
+                        breakdownByMonth: [
+                          { month: '2025-01-01', total: -100 },
+                          { month: '2025-02-01', total: -100 },
+                        ],
+                      },
+                    ],
+                  },
+                ],
+              },
+            ],
+          },
+        }}
+        onCall={mutationSpy}
+      >
+        <PrintTables
+          transactions={[
```

Comment:
```suggestion
      ReportsStaffExpenses: ReportsStaffExpensesQuery;
```

Reply:
I'm not sure why the diff got messed up. The suggestion is for the `GqlMockedProvider` type to change it to rename `StaffExpenseReport` to `ReportsStaffExpenses`. The key there needs to match the name of the GraphQL operation.